assets/model_monitoring/components/model_monitor/model_data_collector_preprocessor/spec.yaml

```diff
@@ -19,6 +19,9 @@ inputs:
   input_data:
     type: uri_folder
     mode: direct
+  extract_correlation_id:
+    type: string
```

Comment:
nit: I think `type: boolean` is supported in yaml. Right?

Reply:
Reference:
https://learn.microsoft.com/en-us/azure/machine-learning/reference-yaml-job-pipeline?view=azureml-api-2#job-inputs

Reply:
I will follow-up